examples/rbac-proxy-resources.yaml

```diff
@@ -0,0 +1,184 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: piraeus-rbac-proxy
+data:
+  piraeus-operator.yaml: |+
+    authorization:
+      resourceAttributes:
+        namespace: linstor
```

Comment:
I'd replace the example namespace with `piraeus`. Also add a comment at the start of the file about the assumed namespaces

Reply:
fixed, thanks